src/server.mjs

```diff
@@ -666,13 +670,32 @@ function getChunkPath(bedFile, dataPath, parsedRegion) {
     }
     if (stringifyRegion(entryRegion) === stringifyRegion(parsedRegion)) {
       // A BED entry is defined for this region exactly
-      if (regionInfo["chunk_path"][i] !== "") {
+      if (regionInfo["chunk"][i] !== "") {
         // And a chunk file is stored for it, so use that.
-        chunkPath = regionInfo["chunk_path"][i];
+        chunkPath = regionInfo["chunk"][i];
         break;
       }
     }
   }
+
+  // if chunk is pulled from an url
+  const chunkDirPath = path.resolve(config.tempDirPath, chunkPath);
+  const baseURLPath = path.resolve(chunkDirPath, "baseURL.txt");
+  // if the chunk directory exists and the baseURL.txt is in it
+  if (fs.existsSync(chunkDirPath) && fs.existsSync(baseURLPath)) {
```

Comment:
I don't think the `baseURLPath` file can exist without the parent directory existing.